sources/packages/backend/apps/db-migrations/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Base Image
+FROM registry.access.redhat.com/ubi8/nodejs-16:1-5
+
+LABEL maintainer="BC GOV"
+
+# Copying package.json and package-lock.json
+COPY ./package*.json ./
```

Comment:
for my info, is it ok to copy in the container's root directory. Shouldn't a `WORKDIR` be specified here?

Reply:
No need at this context we are using the default /opt/app-root/src.